engine/lib/views.php

```diff
@@ -1210,6 +1210,26 @@ function elgg_view_module($type, $title, $body, array $vars = []) {
 	return elgg_view('page/components/module', $vars);
 }
 
+/**
+ * Wrapper function for the message display pattern.
+ *
+ * Box with header, body
+ *
+ * This is a shortcut for {@elgg_view page/components/message}.
+ *
+ * @param string $type  The type of message (main, info, popup, aside, etc.)
```

Comment:
too much copy&paste from elgg_view_module `(main, info, popup, aside, etc.)` whould be `(success, warning, error, etc.)`